.eslintrc.json

```diff
@@ -6,10 +6,6 @@
     "jest": true
   },
   "parser": "@typescript-eslint/parser",
-  "parserOptions": {
```

Comment:
some files werent in the project and it failed for them (i.e. e2e tests setup). So the simplest thing was to remove that.